src/azul/indexer/index_controller.py

```diff
@@ -127,15 +127,15 @@ def contribute(self, event):
                              len(tallies), sum(tally.num_contributions for tally in tallies))
                     for batch in chunked(tallies, self.document_batch_size):
                         entries = [dict(tally.to_message(), Id=str(i)) for i, tally in enumerate(batch)]
-                        self._document_queue.send_messages(Entries=entries)
+                        self._tallies_queue().send_messages(Entries=entries)
             except BaseException:
                 log.warning(f"Worker failed to handle message {message}.", exc_info=True)
                 raise
             else:
                 duration = time.time() - start
                 log.info(f'Worker successfully handled message {message} in {duration:.3f}s.')
 
-    def aggregate(self, event):
+    def aggregate(self, event, retry=False):
         # Consolidate multiple tallies for the same entity and process entities with only one message. Because SQS FIFO
         # queues try to put as many messages from the same message group in a reception batch, a single message per
         # group may indicate that that message is the last one in the group. Inversely, multiple messages per group
```

Comment:
Do you think that **Section 1** description can be removed from its current location and be integrated here. There would be a bit more value since this is were it might actually be used.

Reply:
I agree that the current location isn't a good spot for it. I think moving it to app.py makes the most sense though.